### DISCUSSION TITLE
win11 运行BAT文件闪退

### DISCUSSION THREAD

**Discussion by waloogh** [Rakile/deforumation#19] [Q&A]
win11 运行BAT文件闪退，应该怎么解决？

**Comment by lainollll** [comment]
您可以在命令提示符中手动输入以下命令： "python mediator.py" 并保持该窗口打开，然后输入 "python
deforumation.py" 开始进行反变形。

Den fre 2 juni 2023 18:16waloogh ***@***.***> skrev:

**Reply by waloogh** [reply]
已经使用你最新发布的方法解决了，非常感谢！

**Comment by lainollll** [comment]
👍

Den mån 5 juni 2023 17:09waloogh ***@***.***> skrev:
